packages/core/helpers/ver/4.24.7.ts

```diff
@@ -61,6 +61,7 @@ export const dashboardFiltersMigrate = config => {
   // if there's no dashboardFilters visualization but there are sharedFilters create a visualization and update rows.
 
   config.visualizations = newVisualizations
+  delete config.filterBehavior // deprecated

```

Comment:
Similar to the active property being removed, as long as this is backwards compatible this is fine.

Reply:
I can create my own PR with this. It's basically config cleanup that I forgot to add to the dashboard Filter refactor